adafruit_ht16k33/ht16k33.py

```diff
@@ -45,7 +58,7 @@ def __init__(self, i2c, address=0x70, auto_write=True, brightness=1.0):
         self.blink_rate = 0
         self.brightness = brightness
 
-    def _write_cmd(self, byte):
+    def _write_cmd(self, byte: ReadableBuffer):
```

Comment:
I think we can use `bytearray` for the type here. That is what's listed in the docs for `I2CDevice.write()` Which is what this value gets passed to. https://circuitpython.readthedocs.io/projects/busdevice/en/latest/api.html#adafruit_bus_device.i2c_device.I2CDevice.write